README.md

```diff
@@ -23,17 +23,34 @@ In short: JSON documents with named merkle-links that can be traversed.
 
 ## Repositories
 
-IPLD is a multifaceted, distributed effort. Here are our current repositories:
+IPLD is a multifaceted, distributed effort.
 
+### Specifications
+
+- [specs](https://github.com/ipld/specs) - The specifications for IPLD
 - [cid](https://github.com/ipld/cid) - Self-describing content-addressed identifiers for distributed systems
+
+### Implementations
+
+- [**JavaScript:** js-ipld-dag-cbor](https://github.com/ipld/js-ipld-dag-cbor) - JavaScript implementation of the IPLD spec.
```

Comment:
@diasdavid Those are in this PR already. Are there any other JS modules not in this organization that we need to think of? Have I not categorized these correctly?